src/main/java/com/github/onsdigital/babbage/api/util/SearchUtils.java

```diff
@@ -73,64 +62,60 @@
 public class SearchUtils {
 
     private static final String DEPARTMENTS_INDEX = "departments";
-    private static final String ERRORS_KEY = "errors";
 
     /**
      * Performs search for requested search term against filtered content types and counts contents types.
      * Content results are serialised into json with key "result" and document counts are serialised as "counts".
      * <p>
      * Accepts extra searches to perform along with content search and document counts.
      *
+     * @param searchTerm
```

Comment:
If you think the javadoc is useful/necessary it should be filled out fully - if not remove it completely.